### ISSUE TITLE
powerDNS deployment fails in v40

### ISSUE THREAD

**Issue by gberche-orange** [orange-cloudfoundry/paas-templates#443]
### Expected behavior

PowerDNS concourse deployment shoud succeed

### Observed behavior

```
Task 1116308 | 10:09:36 | Updating instance powerdns-slave-docker: powerdns-slave-docker/ecd06900-768e-4a3e-8840-14a03dde5a59 (0) (00:20:36)
	
                     L Error: 'powerdns-slave-docker/ecd06900-768e-4a3e-8840-14a03dde5a59 (0)' is not running after update. Review logs for failed jobs: mysql, pdns, dnsmasq
	
Task 1116308 | 10:30:12 | Error: 'powerdns-slave-docker/ecd06900-768e-4a3e-8840-14a03dde5a59 (0)' is not running after update. Review logs for failed jobs: mysql, pdns, dnsmasq
```

This was introduced in v40 in commit 7a8d383bdfce4e513f9ed5849ad1b6de40d55242

```
powerdns-slave-docker/ecd06900-768e-4a3e-8840-14a03dde5a59:~# monit summary
The Monit daemon 5.2.5 uptime: 16h 24m

Process 'docker'                    running
Process 'mysql'                     Execution failed
Process 'pdns'                      Execution failed
Process 'dnsmasq'                   not monitored
```

```
cat /var/vcap/sys/log/monit/monit_debugger.dnsmasq_ctl.log:
MONIT-DEBUG dnsmasq_ctl /var/vcap/jobs/containers/bin/ctl start dnsmasq
/var/vcap/jobs/containers/bin/job_properties.sh: line 405: unexpected EOF while looking for matching `"'
```

### Affected release

Reproduced on version 40.x

**Comment by gberche-orange** [comment]
Fixed typo in branch `feature-fix-v41.0.0-RC2`

**Comment by gberche-orange** [comment]
Fixed by adding missing quote in `micro-depls/powerdns-docker/template/openstack-hws/add-system-domain-operators.yml` line 21 

```yaml
  value: INTERNAL_DOMAIN_3=((system_domain))"
```
